src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Utils/ElementSerializer.cs

```diff
@@ -9,8 +9,72 @@
 
 public class ElementSerializer : JsonConverter<List<Element>>
```

Comment:
Side note: It feels weird to deserialize in a class that is called a Serializer. In a separate PR, I'd love to see this renamed to something like `ElementJsonConverter`